### PR TITLE
Add a link to delete team from leave team modal.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/teams/really-leave-team/last-owner.tsx [hunk 1]
```diff
@@ -31,15 +32,20 @@ const ReallyLeaveTeam = (props: Props) => (
         You can't leave the {props.name} team because you're the only owner.
       </Kb.Text>
       <Kb.Text type="Body" center={true} style={styles.bodyText}>
-        You'll have to add another user as an owner before you can leave {props.name}.
+        You'll have to add another user as an owner before you can leave {props.name}, or{' '}
+        <Kb.Text type="BodyPrimaryLink" onClick={props.onDeleteTeam}>
+          delete the&nbsp;team
```
Comment (hyperobject) [inline shared/teams/really-leave-team/last-owner.tsx]: maybe just "delete the team" here

Comment (adamjspooner) [reply shared/teams/really-leave-team/last-owner.tsx]: Do you mean drop the non-breaking space?

Comment (hyperobject) [reply shared/teams/really-leave-team/last-owner.tsx]: yeah, not sure why it's needed?

Comment (adamjspooner) [reply shared/teams/really-leave-team/last-owner.tsx]: Oh, it's to avoid a widow on mobile.

Comment (hyperobject) [reply shared/teams/really-leave-team/last-owner.tsx]: gotcha, sounds good then!

**Pull request merged**